ocs_ci/helpers/dr_helpers.py

```diff
@@ -452,28 +484,39 @@ def wait_for_replication_resources_creation(vr_count, namespace, timeout):
         logger.error(error_msg)
         raise TimeoutExpiredError(error_msg)
 
+    # TODO: Improve the parameter for condition
+    if "cephfs" in namespace:
+        resource_kind = constants.REPLICATION_SOURCE
+        count_function = get_replicationsources_count
+    else:
+        resource_kind = constants.VOLUME_REPLICATION
```

Comment:
Should it be?
```suggestion
        resource_kind = constants.REPLICATIONDESTINATION
```

Reply:
No, resource_kind = constants.VOLUME_REPLICATION for RBD.

Reply:
Ok for rbd